.vscode/settings.json

```diff
@@ -1,9 +1,9 @@
 {
     "jest": {
         "nodeEnv": {
-            "NODE_TLS_REJECT_UNAUTHORIZED": "0", 
-            "PARASOL_MODEL_URL": "https://xxx:443/v1/chat/completions",
-            "EMBEDDING_MODEL_URL": "http://localhost:9999/v1"
+            "PARASOL_MODEL_URL": "http://parasol-chat-trained-predictor.aiworkshop.svc.cluster.local:8080/v1/chat/completions",
```

Comment:
Note this URL will eventually just be `parasol-chat`, not `parasol-chat-trained`

Reply:
Got it